### ISSUE TITLE
Rich Content (A+ Content) blocked by Ad-Blockers

### ISSUE THREAD

**Issue by Rohith-Flix** [easylist/easylist#15367]
Hi Team,

This is to bring to your notice that Rich Content (A+ Content) delivered by Flixmedia on Retailer product pages is being blocked by ad-blockers like Adblock Plus, AdLock, Ghostery, Privacy Badger etc.,.

Flixmedia as a business, helps Brands and Retailers SELL MORE! We deliver rich content (A+ Content) automatically onto retailer product pages through a single line of JavaScript Code. All the rich content is provided to us by the brand directly - content authorized by the brands (like LG, Samsung etc.,)

Here is a live example highlighting the services we deliver,
https://www.harveynorman.com.au/samsung-65-inch-q60a-4k-qled-smart-tv.html

Here is the screenshot highlighting three core services we deliver - HotSpots, MiniSites and INpages,
https://www.awesomescreenshot.com/image/38487207?key=80ea9ccd483c8f96a65bad8425d09e0e

In addition to this we also deliver Augmented Reality (FlixAR) and Shop in Shops (SIS),
Flix AR - https://prnt.sc/k0I4y7pps6S6
https://www.target.com.au/p/lego-reg-technic-ford-mustang-shelby-reg-gt5-reg-42138/66049562

Shop in Shops - https://www.fnac.com/processeurs-amd

All our requests are being blocked by ad-blockers and the services are not able to render as expected.

Could you please review and help in whitelisting our domains for seamless delivery of our services onto Retailer web pages?

Here are the domains that needs to be whitelisted,

1. media.flixcar.com
2. media.flixfacts.com
3. logo.flixfacts.co.uk
4. media.flixsyndication.net
5. t.flix360.com
6. Syndication.flix360.com
7. content.jwplatform.com
8. assets-jpcust.jwpsrv.com
9. ssl.p.jwpcdn.com
10. *.cloudfront.net
12. media.pointandplace.com
13. player.pointandplace.com
14. t.pointandplace.com
15. *.flix360.com
16. *.flix360.io
17. intent://arvr.google.com
18. dev-origin.flixsyndication.net (for staging)
19. dev-delivery.flix360.com (for staging)
The Intent protocol should be whitelisted like this:
https://user-images.githubusercontent.com/105775065/168983608-3eafe62f-3a90-46e2-b4fa-f17a07d016cc.png

Will look forward to have this issue resolved as early as possible.

Best Regards,
Rohith P

**Comment by Rohith-Flix** [comment]
> Aside from this marketing department garbage, and since today isn't April 1st, most of this stuff looks a perfect candidate to be blocked. Specific exception(s) will be added if necessary. Goodbye.

This is so unprofessional way of responding. Is this platform like this, or people here being trained to be like this?

@mapx- from uBO was professional enough and tried to understand the problem in my first ticket and helped in resolving this. For other Ad-blockers, I have been asked to raise a new ticket, for which I have raised this.

Mr or Ms @Khrin, It's quite unfortunate to see the way you respond. Please note that the content we deliver is not marketing department garbage - content that we deliver is the information of the product and is a replica of the Features, Specifications, Videos and other assets available on brand (like Samsung, LG, Sony etc.,.) official website. Brand wants to make sure all the information of the product is syndicated on retailer product page to enahnace shopper browsing experience, increase dwell time and decrease bounce rate.
_Flixmedia helps in increasing dwell time and reducing bounce rate_
And I am least bothered to fool around people on this, see results of content syndication for yourself - [https://www.flixmedia.com/blog/brand-content-boost-add-to-cart-rates](url)
Infact, in many scenario's Retailers themselves have tested to understand the impact of rich A+ content delivered by Flixmedia on their respective product pages, and they clearly see a spike in conversion.

If you do not understand the subject, that is fine, may be you may assign it to someone and stop treating/responding to users rudely.

Thanks for your understanding.

**Comment by Rohith-Flix** [comment]
> 

@MasterKia Thank you for your response. Apologies if any of my words in the issue description have been interpreted as confrontational or bossy.

"Rich Content" or "A+ Content" are most commonly used, so I thought those words would be relevant in understanding.

Let me try to put this in simple words.

**Issue:** Flixmedia delivers product information provided by brand(LG, Samsung, etc.,) automatically onto Retailer (like Harvey Norman, John Lewis, Best Buy etc.,) product pages through a single line of JavaScript code. Product specific information delivered by Flixmedia on Retailer product pages is blocked by Ad-blockers.

Please find the below screenshots/video of the services/information blocked by Ad-blockers,
1) HotSpots, MiniSites and INpages,
https://www.awesomescreenshot.com/image/38487207?key=80ea9ccd483c8f96a65bad8425d09e0e

2) Augmented Reality (FlixAR) and Shop in Shops (SIS),
Flix AR - https://prnt.sc/k0I4y7pps6S6
https://www.target.com.au/p/lego-reg-technic-ford-mustang-shelby-reg-gt5-reg-42138/66049562

Shop in Shops - https://www.fnac.com/processeurs-amd
https://www.awesomescreenshot.com/video/16053026?key=02b8a0f10e80087ac83d913fdc1e3515

Regarding Analytics and Tracking, Flixmedia is very open and transparent concerning the data collected.

Firstly we do not collect any personal information from the website. Secondly as Flixmedia service involves rich assets (Features, 3D, AR, Specifications, Image Gallery, Reviews etc.,.) being ‘called’ from our servers – we always track impressions and clicks and other basic information such as video plays.  This information indeed helps to analyse on the consumer behaviour ONLY for the **content being delivered**.

Our primary goal is to place product information onto retailer product pages which is proven to drive conversion, increase dwell time and reduce bounce rate.

Please let me know if any further details are required.
Sorry once again if any of my words/information above are interpreted as confrontational or bossy.

Thanks.

**Comment by Rohith-Flix** [comment]
> 

@MasterKia Thanks for identifying the issue.
We also have this product information delivered across many other domains.

Please find the below screenshots,
1) https://www.fnac.com/intel
https://www.awesomescreenshot.com/image/38535537?key=ece0b727c87c42d4ea4ca63faeadd3df

2) https://www.boulanger.com/evenement/dyson
https://www.awesomescreenshot.com/image/38535711?key=8f39d0c57daa0b11f9f5c6b2cfecbe33

3) We use JW player to deliver videos from our services - Adblock shows that it is blocking the domain. Could you please check?
https://www.target.com.au/p/lego-reg-technic-ford-mustang-shelby-reg-gt5-reg-42138/66049562
Screenshot: https://prnt.sc/9plXukURQXxH

Kindly need your help in whitelisting the domains which is causing this issue, so that our product information is delivered seamlessly across Retailers.
Your support is highly appreciated!

Thanks.
